packages/web/src/services/remote-config/remote-config-instance.ts

```diff
@@ -18,6 +18,7 @@ export const FEATURE_FLAG_LOCAL_STORAGE_SESSION_KEY = 'featureFlagSessionId-2'
 
 export const remoteConfigInstance = remoteConfig({
   appVersion,
+  platform: 'web',
```

Comment:
One extra thing we could add is differentiating between web and desktop here. Doesn't necessarily have to be in this pr, but that would help when we want to set a min version